CMakeLists.txt

```diff
@@ -361,8 +364,8 @@ endif()
 if(MINGW)
   # suppress warnings about auto imported symbols
   set(US_CXX_FLAGS "-Wl,--enable-auto-import ${US_CXX_FLAGS}")
-  # we need to define a Windows version
-  set(US_CXX_FLAGS "-D_WIN32_WINNT=0x0500 ${US_CXX_FLAGS}")
+  # we need to define a Windows version (Windows Vista)
+  set(US_CXX_FLAGS "-D_WIN32_WINNT=0x0600 -DNTDDI_VERSION=0x06000000 ${US_CXX_FLAGS}")
```

Comment:
I was using the lowest version that contained the necessary API to compile the project successfully (unfortunately, I do not remember which part required this). Officially, Windows 7 can still be our minimum supported version.